kubernetes/test-infra
# kubeadm: update all kind(er) jobs for 1.16

- use prow job annotations
- don't use "latest" kubekins images
- remove 1.12 jobs for kind(er)
- add 1.15 jobs for kind(er)
- don't use "stable" in the names and instead use a version.
existing jobs with "stable" in the name technically should use "ci/latest-x.yy" artifacts
- add kind(er) jobs to release-x.yy-informing dashboards
- update security jobs

/sig cluster-lifecycle
/priority important-longterm
/kind cleanup
/assign @fabriziopandini 
/hold

xref https://github.com/kubernetes/kubeadm/pull/1643
